src/panoptes/pocs/focuser/birger.py

```diff
@@ -308,78 +233,6 @@ def move_by(self, increment):
     # Private Methods
     ##################################################################################################
 
-    def _send_command(self, command, response_length=None, ignore_response=False):
```

Comment:
If the birger needs these error messages for call to `_send_command` then you can just override the method here from the base class, then you don't need to put the parameters in the call each time.

See also the note about the birger specific commands in the base class, which could just be moved back here.